README.md

```diff
@@ -115,3 +115,7 @@ This is currently only available with the `owasp/zap2docker-weekly` or `owasp/za
 See [https://github.com/zaproxy/zaproxy/tree/develop/zap/src/main/dist/lang](https://github.com/zaproxy/zaproxy/tree/develop/zap/src/main/dist/lang) for the full set of locales currently supported.
 
 You can help improve ZAP translations via [https://crowdin.com/project/owasp-zap](https://crowdin.com/project/owasp-zap). 
+
+## Build instructions
+
+Build using `npm run package`
```

Comment:
IMO better add this to other doc, e.g. `BUILDING.md`. The `README.md` is published with the action and this is not relevant for its usage.